webApps/apps.json

```diff
@@ -361,6 +361,63 @@
           "url": "https://github.com/saw-jan/web-app-3dmodel-viewer"
         }
       ]
+    },
+    {
+      "id": "com.github.owncloud.web-app-dicom-viewer",
+      "name": "DICOM Viewer",
+      "subtitle": "Preview DICOM files (.dcm) – both the medical images and their corresponding metadata.",
+      "description": "Blob sources (child-src: 'blob'), data and url connections(connect-src: 'data:' 'https://unpkg.com/detect-gpu@5.0.38/dist/benchmarks/d-intel.json') and unsafe dynamic code evaluation (script-src: '''unsafe-eval''') must all be allowed in CSP",
```

Comment:
@kulmann 
I've added description, please check if it's correct or not.

Reply:
The app store can render markdown for the subtitle and description. Could you add backticks (\`) around the parts in parentheses so that it gets rendered as code? Also, after `and url connections` please add a blank before the opening parenthesis.

Reply:
@kulmann 
I've updated the description.